test/dotnet-new.Tests/GivenThatIWantANewCSApp.cs

```diff
@@ -52,6 +52,21 @@ public void When_dotnet_build_is_invoked_Then_project_builds_without_warnings()
             buildResult.Should().Pass();
             buildResult.Should().NotHaveStdErr();
         }
+
+        [Fact]
+        public void When_dotnet_new_is_invoked_mupliple_times()
```

Comment:
a common pattern we usually use is to include the expectation of the test result in the the name, so for example you could add `_should_fail` here